src/components/LandingPage/AboutEventTile.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from "next/image";
+import type { ReactNode } from "react";
+
+const eventDetailsSectionStyles = "flex flex-col items-center bg-white";
+const eventDetailsContentStyles =
+  "bg-pastel-pink border-[5px] border-dark-pink rounded-[25px] mb-[80px] mt-[50px] md:flex md:justify-center md:max-w-[1100px] md:shadow-[15px_15px_0px_0px_#FF4D6F]";
+const eventImageContainerStyles =
+  "bg-blackish rounded-t-[20px] border-b-[5px] border-dark-pink w-[290px] h-[250px] md:rounded-tr-none md:rounded-l-[20px] md:border-b-0 md:border-r-[5px] md:h-[370px] md:w-[40vw]";
+const eventImageStyles = "w-full h-full object-cover";
+const eventDetailsContainerStyles =
+  "flex items-center w-[290px] h-[370px] rounded-b-[20px] md:w-[50vw] md:rounded-bl-none md:rounded-r-[20px]";
+const eventDetailStyles = "flex items-center ml-[40px]";
+const eventDetailTitleStyles =
+  "text-lg text-blackish font-extrabold leading-[1.2]";
+const iconContainerStyles = "bg-white rounded-[12px] p-[5px] m-[15px]";
+
+const icons = {
+  date: "/svgs/aboutEventTile/date_icon.svg",
+  location: "/svgs/aboutEventTile/location_icon.svg",
+  heart: "/svgs/aboutEventTile/heart_icon.svg",
+  diamond: "/svgs/aboutEventTile/diamond_icon.svg",
+};
+
+interface EventDetailProps {
+  iconSrc: string;
+  iconName: string;
+  children: ReactNode;
+}
+
+const eventDetails = {
+  eventDate: 1731258000,
+  eventLocation: "University of Calgary",
+  eventImg: "/image",
+  eventPrizeAmount: 17000,
+};
+
+const EventDetail = ({ iconSrc, iconName, children }: EventDetailProps) => (
```

Comment:
could we add typing to the parameters
```suggestion
const EventDetail = ({ iconSrc: string, iconName: string, children: ReactNode }: EventDetailProps) => (
```

Reply:
do you want me to replace EventDetailProps with this instead?
const EventDetail = ({ iconSrc, iconName, children }: { iconSrc: string; iconName: string; children: ReactNode; })

because putting the types in the format you specified causes a problem

Reply:
Yes, I would prefer making a props type variable and then using that as the type for props. props:EventsDetailsProps. 

Then breaking the prop apart in the function like {a,b,c} =props;